Index: Makefile
===================================================================
RCS file: /cvs/ports/devel/ruby-bundler/Makefile,v
retrieving revision 1.9
diff -u -r1.9 Makefile
--- Makefile	11 Oct 2014 19:54:46 -0000	1.9
+++ Makefile	1 Jul 2015 18:16:53 -0000
@@ -2,8 +2,7 @@
 
 COMMENT=		ruby application dependency manager
 
-DISTNAME=		bundler-1.3.5
-REVISION =		1
+DISTNAME=		bundler-1.7.15
 CATEGORIES=		devel
 
 HOMEPAGE=		http://gembundler.com/
Index: distinfo
===================================================================
RCS file: /cvs/ports/devel/ruby-bundler/distinfo,v
retrieving revision 1.3
diff -u -r1.3 distinfo
--- distinfo	28 Jun 2013 16:44:23 -0000	1.3
+++ distinfo	1 Jul 2015 18:16:53 -0000
@@ -1,2 +1,2 @@
-SHA256 (bundler-1.3.5.gem) = CLiQR/e4KfPhl6KPsb3nTD9c/qFVL5nfuiN/7jDq/+Q=
-SIZE (bundler-1.3.5.gem) = 268800
+SHA256 (bundler-1.7.15.gem) = v0kbca2v45v3OW0UM5PWsQC9RB2PzwMj1CoTiT4c82s=
+SIZE (bundler-1.7.15.gem) = 308224
Index: pkg/PLIST
===================================================================
RCS file: /cvs/ports/devel/ruby-bundler/pkg/PLIST,v
retrieving revision 1.3
diff -u -r1.3 PLIST
--- pkg/PLIST	28 Jun 2013 16:44:23 -0000	1.3
+++ pkg/PLIST	1 Jul 2015 18:16:53 -0000
@@ -1,13 +1,14 @@
 @comment $OpenBSD: PLIST,v 1.3 2013/06/28 16:44:23 jasper Exp $
 ${GEM_BIN}/bundle${GEM_BIN_SUFFIX}
+${GEM_BIN}/bundler${GEM_BIN_SUFFIX}
 ${GEM_LIB}/cache/${DISTNAME}.gem
 ${GEM_LIB}/gems/${DISTNAME}/
 ${GEM_LIB}/gems/${DISTNAME}/.gitignore
 ${GEM_LIB}/gems/${DISTNAME}/.rspec
 ${GEM_LIB}/gems/${DISTNAME}/.travis.yml
 ${GEM_LIB}/gems/${DISTNAME}/CHANGELOG.md
-${GEM_LIB}/gems/${DISTNAME}/CONTRIBUTE.md
 ${GEM_LIB}/gems/${DISTNAME}/CONTRIBUTING.md
+${GEM_LIB}/gems/${DISTNAME}/DEVELOPMENT.md
 ${GEM_LIB}/gems/${DISTNAME}/ISSUES.md
 ${GEM_LIB}/gems/${DISTNAME}/LICENSE.md
 ${GEM_LIB}/gems/${DISTNAME}/README.md
@@ -16,12 +17,36 @@
 ${GEM_LIB}/gems/${DISTNAME}/bin/
 ${GEM_LIB}/gems/${DISTNAME}/bin/bundle
 ${GEM_LIB}/gems/${DISTNAME}/bin/bundle_ruby
+${GEM_LIB}/gems/${DISTNAME}/bin/bundler
 ${GEM_LIB}/gems/${DISTNAME}/bundler.gemspec
 ${GEM_LIB}/gems/${DISTNAME}/lib/
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler.rb
+${GEM_LIB}/gems/${DISTNAME}/lib/bundler/anonymizable_uri.rb
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/capistrano.rb
+${GEM_LIB}/gems/${DISTNAME}/lib/bundler/cli/
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/cli.rb
+${GEM_LIB}/gems/${DISTNAME}/lib/bundler/cli/binstubs.rb
+${GEM_LIB}/gems/${DISTNAME}/lib/bundler/cli/cache.rb
+${GEM_LIB}/gems/${DISTNAME}/lib/bundler/cli/check.rb
+${GEM_LIB}/gems/${DISTNAME}/lib/bundler/cli/clean.rb
+${GEM_LIB}/gems/${DISTNAME}/lib/bundler/cli/common.rb
+${GEM_LIB}/gems/${DISTNAME}/lib/bundler/cli/config.rb
+${GEM_LIB}/gems/${DISTNAME}/lib/bundler/cli/console.rb
+${GEM_LIB}/gems/${DISTNAME}/lib/bundler/cli/exec.rb
+${GEM_LIB}/gems/${DISTNAME}/lib/bundler/cli/gem.rb
+${GEM_LIB}/gems/${DISTNAME}/lib/bundler/cli/init.rb
+${GEM_LIB}/gems/${DISTNAME}/lib/bundler/cli/inject.rb
+${GEM_LIB}/gems/${DISTNAME}/lib/bundler/cli/install.rb
+${GEM_LIB}/gems/${DISTNAME}/lib/bundler/cli/open.rb
+${GEM_LIB}/gems/${DISTNAME}/lib/bundler/cli/outdated.rb
+${GEM_LIB}/gems/${DISTNAME}/lib/bundler/cli/package.rb
+${GEM_LIB}/gems/${DISTNAME}/lib/bundler/cli/platform.rb
+${GEM_LIB}/gems/${DISTNAME}/lib/bundler/cli/show.rb
+${GEM_LIB}/gems/${DISTNAME}/lib/bundler/cli/update.rb
+${GEM_LIB}/gems/${DISTNAME}/lib/bundler/cli/viz.rb
+${GEM_LIB}/gems/${DISTNAME}/lib/bundler/constants.rb
+${GEM_LIB}/gems/${DISTNAME}/lib/bundler/current_ruby.rb
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/definition.rb
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/dep_proxy.rb
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/dependency.rb
@@ -62,9 +87,15 @@
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/man/gemfile.5
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/man/gemfile.5.txt
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/match_platform.rb
+${GEM_LIB}/gems/${DISTNAME}/lib/bundler/parallel_workers/
+${GEM_LIB}/gems/${DISTNAME}/lib/bundler/parallel_workers.rb
+${GEM_LIB}/gems/${DISTNAME}/lib/bundler/parallel_workers/thread_worker.rb
+${GEM_LIB}/gems/${DISTNAME}/lib/bundler/parallel_workers/unix_worker.rb
+${GEM_LIB}/gems/${DISTNAME}/lib/bundler/parallel_workers/worker.rb
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/psyched_yaml.rb
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/remote_specification.rb
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/resolver.rb
+${GEM_LIB}/gems/${DISTNAME}/lib/bundler/retry.rb
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/ruby_dsl.rb
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/ruby_version.rb
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/rubygems_ext.rb
@@ -83,13 +114,17 @@
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/source/path.rb
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/source/path/installer.rb
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/source/rubygems.rb
+${GEM_LIB}/gems/${DISTNAME}/lib/bundler/source_list.rb
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/spec_set.rb
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/ssl_certs/
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/ssl_certs/.document
+${GEM_LIB}/gems/${DISTNAME}/lib/bundler/ssl_certs/AddTrustExternalCARoot-2048.pem
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/ssl_certs/AddTrustExternalCARoot.pem
-${GEM_LIB}/gems/${DISTNAME}/lib/bundler/ssl_certs/Entrust_net-Secure-Server-Certification-Authority.pem
-${GEM_LIB}/gems/${DISTNAME}/lib/bundler/ssl_certs/GeoTrust_Global_CA.pem
-${GEM_LIB}/gems/${DISTNAME}/lib/bundler/ssl_certs/VerisignClass3PublicPrimaryCertificationAuthority-G2.pem
+${GEM_LIB}/gems/${DISTNAME}/lib/bundler/ssl_certs/Class3PublicPrimaryCertificationAuthority.pem
+${GEM_LIB}/gems/${DISTNAME}/lib/bundler/ssl_certs/DigiCertHighAssuranceEVRootCA.pem
+${GEM_LIB}/gems/${DISTNAME}/lib/bundler/ssl_certs/EntrustnetSecureServerCertificationAuthority.pem
+${GEM_LIB}/gems/${DISTNAME}/lib/bundler/ssl_certs/GeoTrustGlobalCA.pem
+${GEM_LIB}/gems/${DISTNAME}/lib/bundler/ssl_certs/certificate_manager.rb
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/templates/
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/templates/Executable
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/templates/Executable.standalone
@@ -102,6 +137,11 @@
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/templates/newgem/Rakefile.tt
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/templates/newgem/bin/
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/templates/newgem/bin/newgem.tt
+${GEM_LIB}/gems/${DISTNAME}/lib/bundler/templates/newgem/ext/
+${GEM_LIB}/gems/${DISTNAME}/lib/bundler/templates/newgem/ext/newgem/
+${GEM_LIB}/gems/${DISTNAME}/lib/bundler/templates/newgem/ext/newgem/extconf.rb.tt
+${GEM_LIB}/gems/${DISTNAME}/lib/bundler/templates/newgem/ext/newgem/newgem.c.tt
+${GEM_LIB}/gems/${DISTNAME}/lib/bundler/templates/newgem/ext/newgem/newgem.h.tt
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/templates/newgem/gitignore.tt
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/templates/newgem/lib/
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/templates/newgem/lib/newgem/
@@ -115,7 +155,11 @@
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/templates/newgem/test/
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/templates/newgem/test/minitest_helper.rb.tt
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/templates/newgem/test/test_newgem.rb.tt
+${GEM_LIB}/gems/${DISTNAME}/lib/bundler/ui/
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/ui.rb
+${GEM_LIB}/gems/${DISTNAME}/lib/bundler/ui/rg_proxy.rb
+${GEM_LIB}/gems/${DISTNAME}/lib/bundler/ui/shell.rb
+${GEM_LIB}/gems/${DISTNAME}/lib/bundler/ui/silent.rb
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/vendor/
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/vendor/.document
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/vendor/net/
@@ -135,14 +179,18 @@
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/vendor/thor/actions/file_manipulation.rb
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/vendor/thor/actions/inject_into_file.rb
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/vendor/thor/base.rb
+${GEM_LIB}/gems/${DISTNAME}/lib/bundler/vendor/thor/command.rb
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/vendor/thor/core_ext/
-${GEM_LIB}/gems/${DISTNAME}/lib/bundler/vendor/thor/core_ext/dir_escape.rb
-${GEM_LIB}/gems/${DISTNAME}/lib/bundler/vendor/thor/core_ext/file_binary_read.rb
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/vendor/thor/core_ext/hash_with_indifferent_access.rb
+${GEM_LIB}/gems/${DISTNAME}/lib/bundler/vendor/thor/core_ext/io_binary_read.rb
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/vendor/thor/core_ext/ordered_hash.rb
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/vendor/thor/error.rb
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/vendor/thor/group.rb
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/vendor/thor/invocation.rb
+${GEM_LIB}/gems/${DISTNAME}/lib/bundler/vendor/thor/line_editor/
+${GEM_LIB}/gems/${DISTNAME}/lib/bundler/vendor/thor/line_editor.rb
+${GEM_LIB}/gems/${DISTNAME}/lib/bundler/vendor/thor/line_editor/basic.rb
+${GEM_LIB}/gems/${DISTNAME}/lib/bundler/vendor/thor/line_editor/readline.rb
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/vendor/thor/parser/
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/vendor/thor/parser.rb
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/vendor/thor/parser/argument.rb
@@ -156,7 +204,6 @@
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/vendor/thor/shell/basic.rb
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/vendor/thor/shell/color.rb
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/vendor/thor/shell/html.rb
-${GEM_LIB}/gems/${DISTNAME}/lib/bundler/vendor/thor/task.rb
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/vendor/thor/util.rb
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/vendor/thor/version.rb
 ${GEM_LIB}/gems/${DISTNAME}/lib/bundler/vendored_persistent.rb
@@ -175,67 +222,84 @@
 ${GEM_LIB}/gems/${DISTNAME}/man/index.txt
 ${GEM_LIB}/gems/${DISTNAME}/spec/
 ${GEM_LIB}/gems/${DISTNAME}/spec/bundler/
+${GEM_LIB}/gems/${DISTNAME}/spec/bundler/anonymizable_uri_spec.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/bundler/bundler_spec.rb
-${GEM_LIB}/gems/${DISTNAME}/spec/bundler/cli_rspec.rb
+${GEM_LIB}/gems/${DISTNAME}/spec/bundler/cli_spec.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/bundler/definition_spec.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/bundler/dsl_spec.rb
+${GEM_LIB}/gems/${DISTNAME}/spec/bundler/friendly_errors_spec.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/bundler/gem_helper_spec.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/bundler/psyched_yaml_spec.rb
-${GEM_LIB}/gems/${DISTNAME}/spec/bundler/source_spec.rb
+${GEM_LIB}/gems/${DISTNAME}/spec/bundler/retry_spec.rb
+${GEM_LIB}/gems/${DISTNAME}/spec/bundler/settings_spec.rb
+${GEM_LIB}/gems/${DISTNAME}/spec/bundler/source/
+${GEM_LIB}/gems/${DISTNAME}/spec/bundler/source/rubygems_spec.rb
+${GEM_LIB}/gems/${DISTNAME}/spec/bundler/source_list_spec.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/cache/
 ${GEM_LIB}/gems/${DISTNAME}/spec/cache/gems_spec.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/cache/git_spec.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/cache/path_spec.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/cache/platform_spec.rb
+${GEM_LIB}/gems/${DISTNAME}/spec/commands/
+${GEM_LIB}/gems/${DISTNAME}/spec/commands/binstubs_spec.rb
+${GEM_LIB}/gems/${DISTNAME}/spec/commands/check_spec.rb
+${GEM_LIB}/gems/${DISTNAME}/spec/commands/clean_spec.rb
+${GEM_LIB}/gems/${DISTNAME}/spec/commands/config_spec.rb
+${GEM_LIB}/gems/${DISTNAME}/spec/commands/console_spec.rb
+${GEM_LIB}/gems/${DISTNAME}/spec/commands/exec_spec.rb
+${GEM_LIB}/gems/${DISTNAME}/spec/commands/help_spec.rb
+${GEM_LIB}/gems/${DISTNAME}/spec/commands/init_spec.rb
+${GEM_LIB}/gems/${DISTNAME}/spec/commands/inject_spec.rb
+${GEM_LIB}/gems/${DISTNAME}/spec/commands/licenses_spec.rb
+${GEM_LIB}/gems/${DISTNAME}/spec/commands/newgem_spec.rb
+${GEM_LIB}/gems/${DISTNAME}/spec/commands/open_spec.rb
+${GEM_LIB}/gems/${DISTNAME}/spec/commands/outdated_spec.rb
+${GEM_LIB}/gems/${DISTNAME}/spec/commands/package_spec.rb
+${GEM_LIB}/gems/${DISTNAME}/spec/commands/show_spec.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/install/
+${GEM_LIB}/gems/${DISTNAME}/spec/install/binstubs_spec.rb
+${GEM_LIB}/gems/${DISTNAME}/spec/install/bundler_spec.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/install/deploy_spec.rb
+${GEM_LIB}/gems/${DISTNAME}/spec/install/gemfile/
+${GEM_LIB}/gems/${DISTNAME}/spec/install/gemfile/gemspec_spec.rb
+${GEM_LIB}/gems/${DISTNAME}/spec/install/gemfile/git_spec.rb
+${GEM_LIB}/gems/${DISTNAME}/spec/install/gemfile/path_spec.rb
+${GEM_LIB}/gems/${DISTNAME}/spec/install/gemfile_spec.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/install/gems/
 ${GEM_LIB}/gems/${DISTNAME}/spec/install/gems/c_ext_spec.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/install/gems/dependency_api_spec.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/install/gems/env_spec.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/install/gems/flex_spec.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/install/gems/groups_spec.rb
-${GEM_LIB}/gems/${DISTNAME}/spec/install/gems/packed_spec.rb
+${GEM_LIB}/gems/${DISTNAME}/spec/install/gems/mirror_spec.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/install/gems/platform_spec.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/install/gems/post_install_spec.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/install/gems/resolving_spec.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/install/gems/simple_case_spec.rb
+${GEM_LIB}/gems/${DISTNAME}/spec/install/gems/sources_spec.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/install/gems/standalone_spec.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/install/gems/sudo_spec.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/install/gems/win32_spec.rb
-${GEM_LIB}/gems/${DISTNAME}/spec/install/gemspec_spec.rb
-${GEM_LIB}/gems/${DISTNAME}/spec/install/git_spec.rb
-${GEM_LIB}/gems/${DISTNAME}/spec/install/invalid_spec.rb
+${GEM_LIB}/gems/${DISTNAME}/spec/install/gemspecs_spec.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/install/path_spec.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/install/post_bundle_message_spec.rb
+${GEM_LIB}/gems/${DISTNAME}/spec/install/prereleases_spec.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/install/security_policy_spec.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/install/upgrade_spec.rb
-${GEM_LIB}/gems/${DISTNAME}/spec/integration/
-${GEM_LIB}/gems/${DISTNAME}/spec/integration/inject.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/lock/
 ${GEM_LIB}/gems/${DISTNAME}/spec/lock/git_spec.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/lock/lockfile_spec.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/other/
-${GEM_LIB}/gems/${DISTNAME}/spec/other/binstubs_spec.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/other/bundle_ruby_spec.rb
-${GEM_LIB}/gems/${DISTNAME}/spec/other/check_spec.rb
-${GEM_LIB}/gems/${DISTNAME}/spec/other/clean_spec.rb
-${GEM_LIB}/gems/${DISTNAME}/spec/other/config_spec.rb
-${GEM_LIB}/gems/${DISTNAME}/spec/other/console_spec.rb
-${GEM_LIB}/gems/${DISTNAME}/spec/other/exec_spec.rb
+${GEM_LIB}/gems/${DISTNAME}/spec/other/cli_dispatch_spec.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/other/ext_spec.rb
-${GEM_LIB}/gems/${DISTNAME}/spec/other/help_spec.rb
-${GEM_LIB}/gems/${DISTNAME}/spec/other/init_spec.rb
-${GEM_LIB}/gems/${DISTNAME}/spec/other/licenses_spec.rb
-${GEM_LIB}/gems/${DISTNAME}/spec/other/newgem_spec.rb
-${GEM_LIB}/gems/${DISTNAME}/spec/other/open_spec.rb
-${GEM_LIB}/gems/${DISTNAME}/spec/other/outdated_spec.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/other/platform_spec.rb
-${GEM_LIB}/gems/${DISTNAME}/spec/other/show_spec.rb
+${GEM_LIB}/gems/${DISTNAME}/spec/other/ssl_cert_spec.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/quality_spec.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/realworld/
 ${GEM_LIB}/gems/${DISTNAME}/spec/realworld/dependency_api_spec.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/realworld/edgecases_spec.rb
+${GEM_LIB}/gems/${DISTNAME}/spec/realworld/parallel_spec.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/resolver/
 ${GEM_LIB}/gems/${DISTNAME}/spec/resolver/basic_spec.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/resolver/platform_spec.rb
@@ -252,31 +316,36 @@
 ${GEM_LIB}/gems/${DISTNAME}/spec/support/artifice/endopint_marshal_fail_basic_authentication.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/support/artifice/endpoint.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/support/artifice/endpoint_500.rb
+${GEM_LIB}/gems/${DISTNAME}/spec/support/artifice/endpoint_api_forbidden.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/support/artifice/endpoint_api_missing.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/support/artifice/endpoint_basic_authentication.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/support/artifice/endpoint_creds_diff_host.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/support/artifice/endpoint_extra.rb
+${GEM_LIB}/gems/${DISTNAME}/spec/support/artifice/endpoint_extra_api.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/support/artifice/endpoint_extra_missing.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/support/artifice/endpoint_fallback.rb
+${GEM_LIB}/gems/${DISTNAME}/spec/support/artifice/endpoint_host_redirect.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/support/artifice/endpoint_marshal_fail.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/support/artifice/endpoint_redirect.rb
+${GEM_LIB}/gems/${DISTNAME}/spec/support/artifice/endpoint_strict_basic_authentication.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/support/artifice/endpoint_timeout.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/support/builders.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/support/fakeweb/
 ${GEM_LIB}/gems/${DISTNAME}/spec/support/fakeweb/rack-1.0.0.marshal
 ${GEM_LIB}/gems/${DISTNAME}/spec/support/fakeweb/windows.rb
+${GEM_LIB}/gems/${DISTNAME}/spec/support/hax.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/support/helpers.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/support/indexes.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/support/matchers.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/support/path.rb
+${GEM_LIB}/gems/${DISTNAME}/spec/support/permissions.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/support/platforms.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/support/ruby_ext.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/support/rubygems_ext.rb
-${GEM_LIB}/gems/${DISTNAME}/spec/support/rubygems_hax/
-${GEM_LIB}/gems/${DISTNAME}/spec/support/rubygems_hax/platform.rb
+${GEM_LIB}/gems/${DISTNAME}/spec/support/streams.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/support/sudo.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/update/
 ${GEM_LIB}/gems/${DISTNAME}/spec/update/gems_spec.rb
 ${GEM_LIB}/gems/${DISTNAME}/spec/update/git_spec.rb
-${GEM_LIB}/gems/${DISTNAME}/spec/update/source_spec.rb
+${GEM_LIB}/gems/${DISTNAME}/spec/update/path_spec.rb
 ${GEM_LIB}/specifications/${DISTNAME}.gemspec
